peps/tests/test_commands.py

```diff
@@ -6,29 +7,37 @@
 from django.conf import settings
 from django.core import serializers
 from django.core.management import call_command
-from django.core.exceptions import ImproperlyConfigured
 
 import responses
 
 from pages.models import Image
 
-from . import FAKE_PEP_REPO
+from . import FAKE_PEP_ARTIFACT
 
 
-@responses.activate
 class PEPManagementCommandTests(TestCase):
 
-    def setUp(self):
-        self.peps_tar = io.open(FAKE_PEP_ARTIFACT, 'rb')
+    @classmethod
+    def setUpTestData(cls):
+        with io.open(FAKE_PEP_ARTIFACT, 'rb') as fp:
+            peps_tar = fp.read()
         responses.add(
             responses.GET,
             'https://example.net/fake-peps.tar.gz',
             headers={'Last-Modified': 'Sun, 24 Feb 2019 18:01:42 GMT'},
             stream=True,
             content_type='application/x-tar',
             status=200,
-            body=self.peps_tar,
+            body=peps_tar,
         )
+        responses.start()
+        super().setUpTestData()
```

Comment:
Call to `super()` shouldn't be needed here. Did you get an error?

Reply:
👌 No error here